frontend/src/metabase/containers/NewItemMenu/NewItemMenu.unit.spec.tsx

```diff
@@ -43,48 +30,18 @@ type SetupOpts = {
 };
 
 const SAMPLE_DATABASE = createSampleDatabase();
-
-const DB_WITH_ACTIONS = createMockDatabase({
-  id: 2,
-  name: "Postgres with actions",
-  engine: "postgres",
-  native_permissions: "write",
-  settings: { "database-enable-actions": true },
-});
-
-const DB_WITHOUT_WRITE_ACCESS = createMockDatabase({
-  ...DB_WITH_ACTIONS,
-  id: 3,
-  native_permissions: "none",
-});
-
 const COLLECTION = createMockCollection();
 
 async function setup({
-  databases = [SAMPLE_DATABASE, DB_WITH_ACTIONS],
-  hasModels = true,
+  databases = [SAMPLE_DATABASE],
   isEmbeddingIframe,
   entityTypes,
```

Comment:
you should be able to remove this prop and the whole state initialization regarding `entityTypes` on line 54+ as well.

Reply:
Oh, right!
On it. Thanks

Reply:
https://github.com/metabase/metabase/pull/57647/commits/535b702e1c660df799ef95007433a760f57330f5